builtin/providers/aws/resource_aws_route53_record.go

```diff
@@ -507,12 +635,12 @@ func expandRecordName(name, zone string) string {
 	return rn
 }
 
-func resourceAwsRoute53AliasRecordHash(v interface{}) int {
-	var buf bytes.Buffer
-	m := v.(map[string]interface{})
-	buf.WriteString(fmt.Sprintf("%s-", m["name"].(string)))
-	buf.WriteString(fmt.Sprintf("%s-", m["zone_id"].(string)))
-	buf.WriteString(fmt.Sprintf("%t-", m["evaluate_target_health"].(bool)))
-
-	return hashcode.String(buf.String())
```

Comment:
Removed `resourceAwsRoute53AliasRecordHash` because it seems that the default hashing function (schema.HashResource) will handle this fine. I could be reading the code wrong though...